components/docs-view/src/components/CellDiagram.js

```diff
@@ -16,19 +16,319 @@
+        this.loader = React.createRef();
+    }
+
+    componentDidMount = () => {
+        if (this.dependencyGraph.current && this.loader.current) {
+            this.draw();
+        }
+    };
+
+    componentDidUpdate = () => {
+        if (this.network) {
+            this.network.destroy();
+            this.network = null;
+
+            if (this.dependencyGraph.current && this.loader.current) {
+                this.draw();
+            }
+        }
+    };
+
+    componentWillUnmount = () => {
+        if (this.network) {
+            this.network.destroy();
+            this.network = null;
+        }
+    };
+
+    draw = () => {
+        const {data, focusedCell, onClickNode} = this.props;
+        const componentNodes = [];
+        const dataEdges = [];
+        const cellNodes = [];
+        let nodes;
+        let edges;
+        const availableCells = data.cells;
+        const availableComponents = data.components.filter((component) => (focusedCell === component.cell))
+            .map((component) => component);
+
+        const getGroupNodesIds = (group) => {
+            const output = [];
+            nodes.get({
+                filter: function(item) {
+                    if (item.group === group) {
+                        output.push(item.id);
+                    }
+                }
+            });
+            return output;
+        };
+
+        const getDistance = (pts, centroid) => {
+            const cenX = centroid.x;
+            const cenY = centroid.y;
+            const distance = [];
+            let dist = 0;
+            for (let i = 0; i < pts.length; i++) {
+                distance.push(Math.hypot(pts[i].x - cenX, pts[i].y - cenY));
+            }
+            dist = Math.max(...distance);
+            return dist;
+        };
+
+        const getPolygonCentroid = (pts) => {
+            let maxX;
+            let maxY;
+            let minX;
+            let minY;
+            for (let i = 0; i < pts.length; i++) {
+                minX = (pts[i].x < minX || minX === undefined) ? pts[i].x : minX;
+                maxX = (pts[i].x > maxX || maxX === undefined) ? pts[i].x : maxX;
+                minY = (pts[i].y < minY || minY === undefined) ? pts[i].y : minY;
+                maxY = (pts[i].y > maxY || maxY === undefined) ? pts[i].y : maxY;
+            }
+            return {x: (minX + maxX) / 2, y: (minY + maxY) / 2};
+        };
+
+        const getGroupNodePositions = (groupId) => {
+            const groupNodes = getGroupNodesIds(groupId);
+            const nodePositions = this.network.getPositions(groupNodes);
+            return Object.values(nodePositions);
+        };
+
+        if (availableComponents) {
+            availableComponents.forEach((node, index) => {
+                componentNodes.push({
+                    id: node.name,
+                    label: node.name,
+                    shape: "image",
+                    image: "../src/icons/component.svg",
```

Comment:
U can't point to the component this way. This is referring to the image in the source from the build directory. The source will not be there once this is moved in the Cellery installation.

You need to add this image to the public directory and refer using "./component.svg"

Do the same for "focusedCell.svg" and "cell.svg"